bridge/irc_manager.go

```diff
@@ -153,6 +153,11 @@ func (m *IRCManager) HandleUser(user DiscordUser) {
 		}
 	}
 
+	// Don't connect them if we're over our configured connection limit! (Includes our listener)
+	if len(m.ircConnections)+1 >= m.bridge.Config.ConnectionLimit && m.bridge.Config.ConnectionLimit > 0 {
```

Comment:
```suggestion
	if m.bridge.Config.ConnectionLimit > 0 && len(m.ircConnections)+1 >= m.bridge.Config.ConnectionLimit {
```

Do the comparison first, so we don't calculate the length if we don't need to